### PR TITLE
feat: support kill statement

### LINKED ISSUE #533: Support the `KILL` Statement
**What would you like to be added**:

**Why is this needed**:

Support the `KILL` Statement

official document: https://dev.mysql.com/doc/refman/5.6/en/kill.html

### PR DESCRIPTION
**Pull request by Charlie17Li**
Signed-off-by: Charlie17Li <charlie1213800@gmail.com>

**What this PR does**:

support kill statement

**Which issue(s) this PR fixes**:

Fixes #533 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: pkg/runtime/optimize/dal/kill.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+	if len(groups) < 1 {
+		return nil, fmt.Errorf("there is no group")
+	}
+	strs := strings.Split(groups[0], sep)
+	if len(strs) < 2 {
+		return nil, fmt.Errorf("can't resolve the group name: %s", groups[0])
+	}
+	schema := strs[0]
+	format := fmt.Sprintf("%%s_%%0%dd", len(strs[1]))
+	ret.SetDatabase(fmt.Sprintf(format, schema, groupId))
```
Comment (jjeffcaii) [inline pkg/runtime/optimize/dal/kill.go:61]: Use groups item directly, no need to split and format.

File: pkg/util/math/sal_test.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	const (
+		a    = int64(692)
+		b    = int64(3)
+		size = int64(16)
+	)
+
+	res, err := Code(a, size, b)
+	if err != nil {
+		t.Fatal(err)
+	}
```
Comment (jjeffcaii) [inline pkg/util/math/sal_test.go:35]: Use testify package instead.

File: pkg/util/math/sal_test.go [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+	aa, bb := Decode(res, size)
+	if aa != a {
+		t.Errorf(fmt.Sprintf("expect %d, got %d", a, aa))
+	}
```
Comment (jjeffcaii) [inline pkg/util/math/sal_test.go:40]: Same as above.

File: pkg/util/math/sal.go [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+package math
+
+import (
+	"fmt"
+)
+
+const DefaultBase = 16
+
+func Code(num1, base, num2 int64) (int64, error) {
```
Comment (jjeffcaii) [inline pkg/util/math/sal.go:26]: I think it will be better if renaming to`EncodeProcessID`.

**Review by dongzl — APPROVED** [review]

**Review by jjeffcaii — APPROVED** [review]

**Pull request merged**